packages/flutter/test/widgets/gesture_detector_test.dart

```diff
@@ -343,4 +343,67 @@ void main() {
     await longPress(kLongPressTimeout + const Duration(seconds: 1)); // To make sure the time for long press has occurred
     expect(longPressUp, 1);
   });
+
```

Comment:
I would add tests for interactions too, to check for instance that a long press then forced press doesn't trigger the force press callbacks.